app/api/event_invoices.py

```diff
@@ -30,6 +30,8 @@ def query(self, view_kwargs):
         """
         user = current_user
         user_id = view_kwargs.get('user_id')
+        if '/v1/event-invoices' in request.url and not user.is_staff:
```

Comment:
This is not how we do it

Reply:
can you guide me?

Reply:
Check view_args

Reply:
Is the way I opted now is correct or still go for args?